### PR TITLE
fixed metrics (format handling)

### PR DESCRIPTION
**Pull request by mendoxei**

### CODE REVIEW

**Review by aleks-24 — CHANGES_REQUESTED** [review]
Please remove temporary debug print()s and console.log()s before pull requesting the Dev branch.

File: api/project/computation.py [hunk 1]
```diff
@@ -80,8 +80,10 @@ def mock_result(settings):
         for approach in settings['approaches']:
             recommendation = {'approach': approach['name'], 'recommendation': recommend(dataset, approach), 'evals': []}
             for metric in settings['metrics']:
-                evaluation = evaluate_all(dataset['settings'], approach, metric)
-                recommendation['evals'].append({'name': metric['name'], 'evaluation': evaluation})
+                evaluation = evaluate_all(metric['settings'], approach, metric)
+                recommendation['evals'].append(
+                    {'name': metric['name'], 'evaluation': evaluation, 'params': metric['params']})
+                print(metric)
```
Comment (aleks-24) [inline api/project/computation.py:86]: BAD

Comment (mendoxei) [reply api/project/computation.py:86]: <3

Comment (mendoxei) [reply api/project/computation.py:86]: I'll remove it :)

**Review by aleks-24 — APPROVED** [review]

**Pull request merged**